.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+> Please provide a brief summary of the changes made. Please explain why
+> this change was necessary. Was there a problem or an issue this change
+> will address? What will be improved with this change?
```

Comment:
What about using [`html`](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments) comments ? It will avoid having to delete these comments over and over again.